staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -351,10 +351,18 @@ func (o *DrainCmdOptions) deleteOrEvictPods(pods []corev1.Pod) error {
 }
 
 func (o *DrainCmdOptions) evictPods(pods []corev1.Pod, policyGroupVersion string, getPodFn func(namespace, name string) (*corev1.Pod, error)) error {
+	stopCh := make(chan struct{})
```

Comment:
I'm not seeing anything closing or sending to this stopCh... how is this changing behavior?